data_sources/sysmon_eventid_29.yml

```
@@ -0,0 +1,61 @@
+name: Sysmon EventID 29
+id: 06c61e04-2d07-4e85-bcd5-8110938b1b18
+version: 1
+date: '2025-11-14'
+author: Bhavin Patel, Splunk
+description: Data source object for Sysmon EventID 29
+source: XmlWinEventLog:Microsoft-Windows-Sysmon/Operational
+sourcetype: XmlWinEventLog
+separator: EventID
+configuration: https://github.com/SwiftOnSecurity/sysmon-config
+supported_TA:
+- name: Splunk Add-on for Sysmon
+  url: https://splunkbase.splunk.com/app/5709
+  version: 5.0.0
+fields:
+- _time
+- action
+- dest
+- dvc
+- Image
+- EventID
+- EventCode
+- event_type
+- date_hour
+- date_mday
+- date_minute
+- date_month
+- date_second
+- date_wday
+- date_year
+- date_zone
+- User
+- UserID
+- TargetFilename
+- process_id
+- ProcessID
+- Hashes
+- EventRecordID
+- Keywords
+- Channel
+- IMPHASH
+- file_hash
+- file_name
+- file_path
+- severity
+- signature
+- signature_id
+- user
+- user_id
+- SecurityID
+output_fields:
+- Image
+- TargetFilename 
+- file_name 
+- ProcessGuid 
+- Hashes 
+- ProcessID 
+- dest 
+- User 
+- EventCode 
```

I think its best to use CIM values where possible here so that when this is used in the future it work with other DS.


```suggestion
- Image
- TargetFilename 
- file_name
- file_path
- process_guid
- Hashes 
- process_id
- dest 
- User 
- EventCode 
```